.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -10,136 +10,128 @@ assignees: ''
 
 ## Code freeze
 
-- [ ] - 1. Update image-rs to use the latest commit from ocicrypt-rs
+1. - [ ] Update image-rs to use the latest commit from ocicrypt-rs
         
-        * https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml
-          * Change the revision
-          * Run `cargo update -p ocicrypt-rs`
+    * https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml
+      * Change the revision
+      * Run `cargo update -p ocicrypt-rs`
```

Comment:
I had thought that we would be able to skip these steps entirely once we merge the repos. We need to still do this so that the lockfiles are updated? I don't see anywhere that the revision is set in the Cargo file. What does changing the revision refer to?

Reply:
Ok, I guess we can do that in another PR.